cmake/preload/toolchains/util/pico_arm_clang_common.cmake

```
@@ -43,12 +43,30 @@ endforeach()
 
 list(APPEND CMAKE_TRY_COMPILE_PLATFORM_VARIABLES PICO_CLIB)
 
+set(_PICO_CLIB_PATH "${PICO_COMPILER_DIR}/../lib/clang-runtimes/arm-none-eabi")
```

I dunno much about CMake, but I wonder if this block of code could be tidied up by doing e.g.
```suggestion
set(_CLANG_RUNTIMES_DIR "${PICO_COMPILER_DIR}/../lib/clang-runtimes")
set(_PICO_CLIB_PATH "${_CLANG_RUNTIMES_DIR}/arm-none-eabi")
```
etc. ?

---

thanks & done